guides/source/action_cable_overview.md

```diff
@@ -240,9 +240,9 @@ WebNotificationsChannel.broadcast_to(
 ```
 
 The `WebNotificationsChannel.broadcast_to` call places a message in the current
-subscription adapter (Redis by default)'s pubsub queue under a separate
-broadcasting name for each user. For a user with an ID of 1, the broadcasting
-name would be `web_notifications:1`.
+subscription adapter (by default `redis` for production and `async` for development & test)'s 
```

Comment:
would be good also to be clear that we are talking about envs here (e.g for development and test environments)